pkg/health/client/client.go

```diff
@@ -368,47 +388,45 @@ func FormatHealthStatusResponse(w io.Writer, sr *models.HealthStatusResponse, pr
 			localhost = node
 		}
 	}
-	if succinct {
-		fmt.Fprintf(w, "Cluster health:\t%d/%d reachable\t(%s)\n",
-			healthy, len(sr.Nodes), sr.Timestamp)
-		if printAll || healthy < len(sr.Nodes) {
-			fmt.Fprintf(w, "  Name\tIP\tNode\tEndpoints\n")
-		}
-	} else {
-		fmt.Fprintf(w, "Probe time:\t%s\n", sr.Timestamp)
-		fmt.Fprintf(w, "Nodes:\n")
-	}
+
+	fmt.Fprintf(w, "Cluster health:\t%d/%d reachable\t(%s)\n",
+		healthy, len(sr.Nodes), sr.Timestamp)
 
 	if localhost != nil {
-		formatNodeStatus(w, localhost, printAll, succinct, verbose, true)
-		maxLines--
+		printed := formatNodeStatus(w, localhost, allNodes, verbose, true)
```

Comment:
nit the var declaration is not necessary i.e if formatNodeStatus() {}

Reply:
Very valuable recommendation that makes the code even easier. Implemented.